tools/integration-tests/tester/framework/framework.go

```diff
@@ -320,26 +336,28 @@ func (f *Framework) CreateDRNGNetwork(name string, members, peers, minimumNeighb
 // CreateNetworkWithMana creates and returns a (Docker) Network that contains peers that all have some mana.
 // Mana is gotten by sending faucet requests.
 func (f *Framework) CreateNetworkWithMana(name string, peers, minimumNeighbors int, config CreateNetworkConfig) (*Network, error) {
-	n, err := f.CreateNetwork(name, peers, minimumNeighbors, config)
-	if err != nil {
-		return nil, err
-	}
 	if !config.Faucet {
 		return nil, fmt.Errorf("faucet is required")
 	}
 	if !config.Mana {
 		return nil, fmt.Errorf("mana plugin is required to load mana snapshot")
 	}
+
+	n, err := f.CreateNetwork(name, peers, minimumNeighbors, config)
+	if err != nil {
+		return nil, err
+	}
+	time.Sleep(15 * time.Second)
```

Comment:
The sleep was added, what is its purpose? The `CreateNetwork` waits for successful peering, doesn't this also assure that the every node should be running?